scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -69,36 +67,40 @@ workflow GvsBulkIngestGenomes {
             vcf_index_files_column_name = vcf_index_files_column_name,
     }
 
-    call PrepareBulkImport.GvsPrepareBulkImport as PrepareBulkImport {
+    call GenerateImportFofnFromDataTable {
         input:
-            project_id = GetWorkspaceName.workspace_namespace,
+            google_project_id = GetWorkspaceName.workspace_namespace,
```

Comment:
nit: maybe a better name would be `terra_google_project_id` or `workspace_google_project`?